testing/checkpolicy/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Tycho Andersen <tycho@docker.com>
+pkgname=checkpolicy
+pkgver=2.6
+pkgrel=0
+pkgdesc="SELinux policy checker"
+url="http://www.selinuxproject.org"
+arch="all"
+license="GPLv2"
+depends="flex-dev"
```

Comment:
Well, it seems not:

```
alpine-edge:~/aports/testing/checkpolicy/pkg/checkpolicy/usr/bin$ ldd checkmodule 
	/lib/ld-musl-x86_64.so.1 (0x55736fbc8000)
	libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x55736fbc8000)
alpine-edge:~/aports/testing/checkpolicy/pkg/checkpolicy/usr/bin$ ldd checkpolicy 
	/lib/ld-musl-x86_64.so.1 (0x5624657fd000)
	libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x5624657fd000)
```

However, the build system passes -lfl and libfl.so lives in flex-dev, so I just assumed it did. Anyway, I took it out and addressed the rest of your comments. Thanks!